convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -60,16 +70,41 @@ def run(self):
         logger.task("Convert: Remove excluded packages")
         logger.info("Searching for the following excluded packages:\n")
         try:
-            pkgs_to_remove = pkghandler._get_packages_to_remove(system_info.excluded_pkgs)
-            pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove)
+            pkgs_to_remove = sorted(pkghandler._get_packages_to_remove(system_info.excluded_pkgs))
+            pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove))
```

Comment:
This will fail if the `remove_pkgs_unless_from_redhat` returns None (which could happen). 

A fix for that is implemented in https://github.com/oamg/convert2rhel/pull/888/files#diff-7bab0f1619d739dd7e18e3643a6814d444189bb25fc420206ae8e3a0b3ddde85R85, maybe we can get away in this PR by having the fix in the next one?

Reply:
I put a fix for this in the latest commit, remove_pkgs_unless_from_redhat will return an empty list if the pkgs_to_remove is empty